test-configs.yaml

```diff
@@ -1677,46 +1677,57 @@ test_configs:
   - device_type: imx6q-sabresd
     test_plans:
       - baseline
+      - baseline-nfs
 
   - device_type: imx6qp-sabresd
     test_plans:
       - baseline
+      - baseline-nfs
```

Comment:
Results not found on staging.

It seems to be an issue with a timeout while downloading the rootfs tarball:
https://lavalab.nxp.com/scheduler/job/96175

Could you please resubmit manually with a longer timeout to confirm?

If this is the case, we can extend the timeout in the KernelCI job template.

Reply:
Tested OK on staging: https://staging.kernelci.org/test/plan/id/5f902c67a2dd60d867729032/